repos/sourcelink.proj

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PreferredHash>51310e65e75010467993f793e1739d1a1dad50c5</PreferredHash>
+    <SourceDirectory>sourcelink</SourceDirectory>
```

Comment:
Doesn't `SourceDirectory` default to the proj name? I wouldn't expect to need a custom name here because there are no special characters.

Reply:
I had it for consistency. I can try to remove it and see if it picks up without explicitly assigning it the value.

Reply:
Here's some precedent for the default, FWIW:

https://github.com/dotnet/source-build/blob/3a522d7ee800df9a01bd9272445e21431c256eb8/repos/corefx.proj#L3-L7

https://github.com/dotnet/source-build/blob/3a522d7ee800df9a01bd9272445e21431c256eb8/repos/core-setup.proj#L3-L7

https://github.com/dotnet/source-build/blob/3a522d7ee800df9a01bd9272445e21431c256eb8/repos/arcade.proj#L3-L7